osu.Game/Online/API/Requests/Responses/APIBeatmapSet.cs

```diff
@@ -125,6 +125,9 @@ public string AuthorString
         [JsonProperty(@"beatmaps")]
         public APIBeatmap[] Beatmaps { get; set; } = Array.Empty<APIBeatmap>();
 
+        [JsonProperty(@"converts")]
+        public APIBeatmap[] Converts { get; set; } = Array.Empty<APIBeatmap>();
```

Comment:
I kinda think this field should be marked nullable and have a default of `null` As per [API docs](https://osu.ppy.sh/docs/index.html#beatmapsetcompact), it is an optional attribute on `BeatmapsetCompact` and there is no guarantee that there is actually valid data for reading here.

This sort of applies to `Beatmaps`, too, but let's not go off course too much. I think this particular case is a bigger issue anyhow, as there are actual other contexts in the game where trying to get valid data out of `Converts` is just straight up wrong as the property is just not returned (beatmap listing, for one).